HW0/P6/P6B.py

```diff
@@ -16,16 +17,27 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(-i) for i in xrange(7)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+
+        # Serial
+        start = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
@kevinkeyjkw That is what I did as well. The way it is currently written, `pool.map(burnTime(t), range(N))` actually computes `burnTime(t)` *once* before `pool.map` is called, effectively doing something like `pool.map(1.0, range(N))`, I believe. So I think your scaling plot is wrong, and it should level out around 4 (since there are 4 processes in your pool). Also, your `# Parallel` and `# Serial` comments are switched. :)